app/ide-desktop/lib/client/src/security.ts

```diff
@@ -8,7 +8,13 @@ import * as electron from 'electron'
 // =================
 
 /** The list of hosts that the app can access. They are required for user authentication to work. */
-const TRUSTED_HOSTS = ['accounts.google.com', 'accounts.youtube.com', 'github.com']
+const TRUSTED_HOSTS = [
+    'accounts.google.com',
+    'accounts.youtube.com',
+    'github.com',
+    'production-enso-domain.auth.eu-west-1.amazoncognito.com',
+    'pb-enso-domain.auth.eu-west-1.amazoncognito.com',
```

Comment:
Are we able to somehow build these cognito links out of the environment names? its basically `${env}-enso-domain.auth.eu-west-1.amazoncognito.com` and I would love to omit hardcoding

Reply:
i'm not sure unfortunately - note that dev env uses `pbuchu` instead of `pb` but that can be easily changed.
the bigger issue is that the environment names are not available in `lib/client` (the electron app) as the dashboard is not imported at all, and i'd prefer a little bit of hardcoding rather than bringing in an entire dependency for one constant

actually another issue is, i don't think we have an array of environment names anywhere.
we can consider moving environment names to `common/` though, and import in both `client/` and `dashboard`?